imports/components/@primitives/layout/global/index.js

```diff
@@ -160,50 +173,52 @@ class GlobalWithoutData extends Component {
       "/stories/",
     ];
 
-    if (Meteor.isCordova) {
-      document.addEventListener("click", linkListener);
-      document.addEventListener("deviceready", () => {
-        universalLinks.subscribe(null, ({ path }) => {
-          this.setState({ universalLinkLoading: true });
-          let isQueryRoute = false;
-          queryRoutes.forEach((url) => {
-            if (path.includes(url)) isQueryRoute = true;
-          });
-          if (isQueryRoute) {
-            // const pathArray = path.split("/").filter(Boolean);
-            // const section = pathArray[0];
-            // const urlTitle = pathArray[1];
-            // this.props.client.query({ query, variables: { urlTitle })
-            //   .then(({ data: { somehwere: id } }) => {
-            //     this.go(`/#{this.section}/);
-            //   })
-            // this.go(`/${section}`);
-            alert("this is a query route");
-            return;
-          }
-          this.go(path);
-        });
-      }, false);
+    this.state = ({ universalLinkLoading: true });
```

Comment:
we should never mutate the state directly. React will ignore it and overwrite it. Instead we should use `setState`.